app/controllers/api/v1/passwords_controller.rb

```diff
@@ -0,0 +1,8 @@
+module Api
+  module V1
+    class PasswordsController < DeviseTokenAuth::PasswordsController
+      include Api::Concerns::ActAsApiRequest
+      skip_before_action :check_json_request, on: :edit
```

Comment:
Why was this necessary?

Reply:
I understand that edit request in a link, does not include a json on the request.